scripts/apps/authoring/authoring/services/AuthoringWorkspaceService.js

```diff
@@ -44,6 +44,11 @@ export function AuthoringWorkspaceService($location, superdeskFlags, authoring,
             }
         }.bind(this);
 
+        if (item._type === 'externalsource' && config.features && 'editFeaturedImage' in config.features &&
```

Comment:
would be good to add a comment here or put this into a function to give it a name, it's clear from PR name what it does but not so much from the code